packages/commonwealth/client/styles/components/component_kit/cw_table.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../shared';
+
+.Table {
+  table{
+    border-collapse: collapse;
+
+    thead {
+      border-top: 1px solid $neutral-200;
+      border-bottom: 1px solid $neutral-200;
+      background-color: $neutral-50;
+
+      tr {
+        th {
+          div.header-content {
```

Comment:
everywhere where you put html element tag prefix before the class eg. `div`, `span` - this is not necessary. Class selector is enough in this case. Please remove